geopandas/tests/test_geodataframe.py

```diff
@@ -372,6 +368,20 @@ def test_transform(self):
         utm = lonlat.to_crs(epsg=26918)
         self.assertTrue(all(df2['geometry'].geom_almost_equals(utm['geometry'], decimal=2)))
 
+    def test_to_crs_geo_column_name(self):
+        """ Test to_crs() with different geometry column name (GH#339) """
```

Comment:
it's better to not use triple quote comments but just use `#`, as otherwise testing frameworks (or at least nose does this) takes the docstring as the displayed name instead of the function name (and if you see a test is failing, you rather want to know the function name)